src/app.js

```diff
@@ -36,6 +38,11 @@ export default class App extends React.Component {
         view: { path }
       })
     })
+    window.addEventListener('beforeunload', () => {
+      const { view, flashcards } = this.state
```

Comment:
The browser's history (back/forward) will keep track of the view, so you don't need to persist it.